test/e2e/ginkgo/README.md

```diff
@@ -0,0 +1,25 @@
+# Kubestellar end to end testing
+The end to end testing includes:
+1. deployments are downsync propegated to the WECs.
+1. Update of the workload object on WDS should update the object on the WECs. Increase the number of replicas from 1 to 2, verify they are updated on the WECs.
+1. Changing the bindingpolicy objectSelector to no longer match should delete the object from the WECs.
+1. Changing the bindingpolicy objectSelector to match should create the object on the WECs.
+1. Delete of the bindingpolicy object should delete the object on the WECs.
+1. Delete of the overlapping bindingpolicy object should not delete the object on the WECs.
+1. Delete of the workload object on WDS deletes the relevant objects on the WECs.
+1. Delete of a bindingpolicy deletes the relevant objects on the WECs.
+1. Singleton status update
+1. Object field filtering for services
```

Comment:
I think we usually describe this as "cleaning".

What about Job objects?

Reply:
You are right.  I think the name filtering is more appropriate, but I'll switch it back to cleaning.

Reply:
Filtering is not bad, but "field filtering" is too specific. Cleaning involves removing labels too, for example. "content filtering" would work.